src/screens/Assets/AssetsList.js

```diff
@@ -369,9 +373,9 @@ class AssetsList extends React.Component<Props, State> {
       .map(id => assets[id])
       .map(({ symbol, ...rest }) => ({
         symbol,
-        balance: getBalance(balances, symbol),
         paymentNetworkBalance: getBalance(paymentNetworkBalances, symbol),
         ...rest,
+        balance: getBalance(balances, symbol),
```

Comment:
this was moved here because `...rest` for btcAsset unfortunately had a balance: 0 default which overrides the real balance.